cmd/generate-inventory/main.go

```diff
@@ -220,6 +220,7 @@ func main() {
 
 	funcsMap := sprig.FuncMap()
 	funcsMap["pascalcase"] = PascalCase
+	funcsMap["snakecase"] = SnakeCase
```

Comment:
Not sure, this is actually needed, since the value in the config file for generate-inventory are snake case by convention.